package.json

```diff
@@ -22,11 +22,11 @@
   "homepage": "https://github.com/praekelt/go-ndoh",
   "dependencies": {
     "vumigo_v02": "~0.2",
-    "q": "~0.9",
+    "q": "~0.9.7",
     "lodash": "~2.4.1",
-    "moment": "~2.6.0",
-    "go-jsbox-metrics-helper": "~0.1.1",
-    "libxmljs": "0.8.x"
+    "moment": "~2.6",
+    "go-jsbox-metrics-helper": "^0.1.0",
+    "libxmljs": "~0.8.1"
```

Comment:
What's with all the version changes, and more specifically, why did `go-jsbox-metrics-helper` go back to `0.1.0`?